core/src/dird/ua_status.cc

```diff
@@ -513,46 +532,53 @@ static bool DoSubscriptionStatus(UaContext* ua)
     return false;
   }
 
+  char now[30] = {0};
+  bstrftime(now, sizeof(now), (utime_t)time(NULL), "%F %T");
+
   if (kw_all || kw_detail) {
+    ua->send->ObjectKeyValue(
+        "version", "Bareos version: ", kBareosVersionStrings.Full, "%s");
+    ua->send->ObjectKeyValue("os", kBareosVersionStrings.GetOsInfo(),
+                             " (%s)\n");
+    ua->send->ObjectKeyValue("binary-info",
+                             "Binary info: ", kBareosVersionStrings.BinaryInfo,
+                             "%s\n");
+    ua->send->ObjectKeyValue("date", "Date: ", now, "%s\n");
     ua->SendMsg(_("\nDetailed backup unit report:\n"));
-    ua->send->ObjectStart("unit-detail");
     ua->db->ListSqlQuery(
         ua->jcr,
         BareosDb::SQL_QUERY::subscription_select_backup_unit_overview_0,
-        ua->send, HORZ_LIST, true);
-    ua->send->ObjectEnd("unit-detail");
+        ua->send, HORZ_LIST, "unit-detail", true);
   }
   if (kw_all || kw_detail || !kw_unknown) {
     ua->SendMsg(_("\nBackup unit totals:\n"));
-    ua->send->ObjectStart("total-units-required");
-
     PoolMem query(PM_MESSAGE);
     ua->db->FillQuery(
         query, BareosDb::SQL_QUERY::subscription_select_backup_unit_total_1,
         me->subscriptions);
-    ua->db->ListSqlQuery(ua->jcr, query.c_str(), ua->send, VERT_LIST, true);
-    ua->send->ObjectEnd("total-units-required");
+    ua->db->ListOneRowSqlQuery(ua->jcr, query.c_str(), ua->send, VERT_LIST,
+                               "total-units-required", true);
+    std::string signature_source
+        = get_subscription_status_signature_source_text(ua, now);
+    std::string hash = compute_hash(signature_source.c_str());
+    ua->send->ObjectKeyValue("signature", "Signature: ", hash.c_str(), "%s\n");
```

Comment:
I really wouldn't call it a signature, it is just a hash or checksum, so we should probably name it as such.

Reply:
I named it Checksum now.